tfcoreml/_layers.py

```diff
@@ -860,6 +860,24 @@ def square(op, context):
   context.builder.add_elementwise(
       output_name, [input_name, input_name], output_name, 'MULTIPLY')
 
+def pow(op, context):
+
+  input_name = compat.as_bytes(op.inputs[0].name)
+  output_name = compat.as_bytes(op.outputs[0].name)
+
+  context.translated[output_name] = True
+  context.builder.add_unary(
+    output_name, input_name, output_name, 'power')
```

Comment:
I guess you also want to read the value of alpha from TF and set it for CoreML (i.e. y = x^alpha).

Reply:
Oh I've got no idea how I missed that. Will fix and add tests. Thanks!